packages/expo-router/src/utils/stack.ts

```diff
@@ -11,7 +11,7 @@ export function isRoutePreloadedInStack(
   if (!navigationState || navigationState.type !== 'stack') {
     return false;
   }
-  return (navigationState as StackNavigationState<ParamListBase>).preloadedRoutes.some(
+  return (navigationState as StackNavigationState<ParamListBase>).preloadedRoutes?.some(
```

Comment:
When is it neede?

Reply:
I'll see if I can get you a repro, I'm currently getting this on all link previews in my dogfood app.